ui/src/app/machines/views/MachineDetails/MachineSummary/OverviewCard/DetailsCard/DetailsCard.tsx

```diff
@@ -50,10 +50,10 @@ const DetailsCard = ({ machine }: Props): JSX.Element => {
           {machine.domain?.name}
         </span>
       </div>
-      {machine.pod ? (
-        <div>
-          <div className="u-text--muted">Host</div>
-          <span data-test="host">
+      <div>
+        <div className="u-text--muted">Host</div>
+        <span data-test="host">
+          {machine.pod ? (
```

Comment:
I've changed the Host section to always render, regardless of whether the machine exists in a host or not (renders "None" instead of a host name). Otherwise it gets a bit spaghetti-y trying to handle responsive layouts with an unknown number of columns.